lib/hiera/scope.rb

```diff
@@ -79,5 +82,9 @@ def find_hostclass(scope)
       end
     end
     private :find_hostclass
+
+    # This is needed for type conversion to work
+    def_delegators :@real, :call_function
```

Comment:
How is this different than the definitions of catalog(), resource(), and compiler() in this class?  Should all of those be converted to forwards also?

Reply:
It should not be different, and the existing could probably be changed. I did not want to change more than what was required though in case there is some subtle difference as this is for a .z release.

Reply:
👍